Transfer/Sources/CreateTransferController.swift

```diff
@@ -376,7 +378,13 @@ extension CreateTransferController: CreateTransferView {
 
     func showError(_ error: HyperwalletErrorType, pageName: String, pageGroup: String, _ retry: (() -> Void)?) {
         let errorView = ErrorView(viewController: self, error: error, pageName: pageName, pageGroup: pageGroup)
-        errorView.show(retry)
+        print("show the error group *************", error.group)
+        if error.group == .business && buttonSelection == true {
+            errorView.show(retry)
+        } else if error.group != .business {
+            errorView.show(retry)
```

Comment:
what's the difference in if and else logic? Also handle this in Presenter

Reply:
ok